pkg/util/mount/mount.go

```diff
@@ -33,6 +33,14 @@ const (
 	MountsInGlobalPDPath = "mounts"
 )
 
+type MountPathType string
```

Comment:
Isn't it just FileType? What's "type of mount path"? Does it check only mount paths? Mount paths are always directories, right?

Reply:
> Mount paths are always directories, right?

Mount path can be single file path.

Reply:
yes, still `GetMountPathType` tells type of a file represented by a path, not what type of mount it is. It is not mounted yet.